DEVELOPMENT_GUIDE.md

```diff
@@ -0,0 +1,248 @@
+
+```bash
+nvm install lts/iron
+```
+This will fetch, per the list above, node `v20.12.0`.
+
+```bash
+➜ nvm install lts/iron
+Downloading and installing node v20.12.0...
+Downloading https://nodejs.org/dist/v20.12.0/node-v20.12.0-darwin-arm64.tar.xz...
+############################################################################################################################################################ 100.0%
+Computing checksum with sha256sum
+Checksums matched!
+Now using node v20.12.0 (npm v10.5.0)
+(mlflow-dev-env)
+```
+
+Once you have node installed, you can activate this new version by running:
+
+```bash
+➜ nvm use v20.12.0
+Now using node v20.12.0 (npm v10.5.0)
+```
+
+### Yarn setup
+
+With the above steps completed, nvm will have acquired both node and `npm` (The Node package manager).
+
+We'll use `npm` to install `yarn` so that we can do active development, build a static version of the website, and compile the blog markdown content into renderable `.tsx` files.
+
+```bash
+npm install --global yarn
+```
+
+## Building your development environment
+
+To fetch and build the development environment for Docusaurus, simply navigate, from the root of your fork:
+
+```bash
+cd website
+```
+
+From this directory, run the following command:
+
+```bash
+yarn
+```
+
+This will fetch the required packages and ensure that your environment is up to date with required dependencies to build, compile, or start an interactive environment.
+
+## Development
+
+There are two primary activities that can be worked on in this repository:
+
+- New page development or updates to existing main pages
+- Blog writing
+
+The recommended development process for validating changes for these two different workstreams are quite different.
+
+### Core site development
+
+When working on new pages or updating existing pages within the website, it can beneficial to use the live viewer mode for Docusaurus.
+In this mode, `yarn` will start a local server that will respond to the state of the files from your development branch, dynamically updating
+a live view of the site within a browser window.
+
+To enable live preview, simply run:
+
+```bash
+yarn start
+```
+
+This will initialize the local development server:
+
+```bash
+➜ yarn start
+yarn run v1.22.19
+$ docusaurus start
+
+    ---------------------------------------------------------------------------------------------------------------------------
+
+                                                  Update available 3.0.1 → 3.1.1
+
+                        To upgrade Docusaurus packages with the latest version, run the following command:
+        `yarn upgrade @docusaurus/core@latest @docusaurus/plugin-client-redirects@latest @docusaurus/preset-classic@latest
+                   @docusaurus/module-type-aliases@latest @docusaurus/tsconfig@latest @docusaurus/types@latest`
+
+    ---------------------------------------------------------------------------------------------------------------------------
```

Comment:
```suggestion
...
```

Reply:
I'd remove the output section. It doesn't seem important.

Reply:
yep deleted all of that and just mentioned that a local url can be used for active dev